File: Modul-3/Soal 1/PRAK301-2410817320015-NazwaAmanda.c
#include <stdio.h>

    int main(){
    int angka1, angka2, angka3; scanf("%d %d %d",&angka1 ,&angka2, &angka3);
    if(angka1 > angka2 && angka1 > angka3){
        if(angka2 > angka3) {printf("%d %d %d",angka3 , angka2, angka1);}
        else {printf("%d %d %d",angka2, angka3, angka1);} }

    else if (angka2 > angka3 && angka2 > angka1){
        if(angka1 > angka3) {printf("%d %d %d",angka3, angka1, angka2);}
        else{printf("%d %d %d", angka1, angka3, angka2);}}
    
    else if(angka3 > angka1 && angka3 > angka2){
        if(angka1 > angka2) {printf("%d %d %d",angka2,angka1, angka3);}
        else {printf("%d %d %d",angka1, angka2, angka3);}}
        return 0;
}

File: Modul-3/Soal 1/PRAK301-2410817320015-NazwaAmanda.py
angka1, angka2, angka3 = map(int, input().split()) 
if angka1 >= angka2 >= angka3 : print(f"{angka3} {angka2} {angka1}")
elif angka1 >= angka3 >= angka2 : print(f"{angka2} {angka3} {angka1}")

elif angka2 >= angka3 >= angka1 : print(f"{angka1} {angka3} {angka2}")
elif angka2 >= angka1 >= angka3 : print(f"{angka3} {angka1} {angka2}")

elif angka3 >= angka1 >= angka2 : print(f"{angka2} {angka1} {angka3}")
elif angka3 >= angka2 >= angka1 : print(f"{angka1} {angka2} {angka3}")